### ISSUE TITLE
kindoff a lily pad bug

### ISSUE THREAD

**Issue by Potshott** [SanAndreasP/ClaySoldiersMod#244]
[ Please replace text in [] with the answer from the above question! ]

Do you use Java 8?
yes

What version of Minecraft do you use?
1.12.2

What version of MinecraftForge do you use?
14.23.4.2757

What version of SanLib do you use?
1.5.0

What version of Clay Soldiers do you use?
1.12.2 beta
 
What is the issue / bug you have encountered? Give us a short description.
if a soldiers has a lily pad upgrade and swims under a lily pad they get stuck

How can it be reproduced?
have a clay soldier with a lilypad and put it under a lily pad

Please post your log file (either in /logs/fml-client-latest.log or /logs/latest.log) on https://gist.github.com and provide the link:
[Link to log file]
![2018-11-03_13 14 55](https://user-images.githubusercontent.com/44125490/47952678-f7b28780-df6a-11e8-8302-28a546f2dad6.png)

**Comment by SanAndreasP** [comment]
get stuck as in... not move at all?
The new update has had some AI changes. Can you test if your issue is still valid?

**Comment by Potshott** [comment]
Yes they still stay there even with new update. They actually do change direction now but not move forward or not while I watched them

**Comment by Potshott** [comment]
update: they can escape after a while but i can take ages and if you use a upgrade to lure them out you have to have it just at the edge of the lilypad

**Comment by Potshott** [comment]
![2019-08-28_20 45 33](https://user-images.githubusercontent.com/44125490/63887638-25266100-c9d5-11e9-95d0-7efd6d42d26e.png)
happen with turtels too
